grammars/git commit message.cson

```diff
@@ -9,33 +9,35 @@
 'patterns': [
   {
     'begin': '\\A(?!#)'
-    'end': '(?=# Please enter the commit message)'
+    'end': '^(?<!\\A)(?=# Please enter the commit message)'
     'name': 'meta.scope.message.git-commit'
     'patterns': [
       {
-        'begin': '\\G'
-        'end': '^(?!\\G)'
-        'patterns': [
-          {
-            'captures':
-              '1':
-                'name': 'keyword.other.$2.git-commit'
-            'match': '\\G((fixup|squash)!)\\s*'
-          }
-          {
-            'match': '.{66,}$'
-            'name': 'invalid.illegal.line-too-long.git-commit'
-          }
-          {
-            'match': '.{51,}$'
-            'name': 'invalid.deprecated.line-too-long.git-commit'
-          }
-        ]
+        'captures':
+          '1':
+            'name': 'keyword.other.$2.git-commit'
+        'match': '\\G((fixup|squash)!)\\s*'
```

Comment:
[I wasn't the one who added that](https://github.com/atom/language-git/blob/master/grammars/git%20commit%20message.cson#L20-L23)... but I can fix that if you prefer.